frontend.yml

```diff
@@ -19,6 +19,11 @@ objects:
         manifestLocation: "/apps/chrome/js/fed-mods.json"
         config:
           ssoUrl: ${SSO_URL}
+        modules:
+          - id: 'satellite-token'
```

Comment:
@Hyperkid123 I noticed this is duplicated with my changes in chrome-service-backend: https://github.com/RedHatInsights/chrome-service-backend/pull/668. I know this is used for the frontend-operator - but can you explain why we need both? Thanks.

Reply:
We don't need both. Only the chrome service. This will become relevant and the source of truth once we have the new FEO features.